test/cli/run/env.test.ts

```diff
@@ -338,3 +338,12 @@ test(".env in a folder doesn't throw an error", () => {
   const { stdout } = bunRun(`${dir}/index.ts`);
   expect(stdout).toBe("hey");
 });
+
+test("#3911", () => {
```

Comment:
Can you add a test for a bounds check here? That if it’s \\n it doesn’t go out of bounds

Reply:
sure